src/pages/version/Banners/ErrorBanner/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import Callout from "@leafygreen-ui/callout";
+import { DisplayModal } from "components/DisplayModal";
+import Icon from "components/Icon";
+import {
+  BannerContainer,
+  TitleWrapper,
+  OrderedList,
+  ListItem,
+  ModalTrigger,
+} from "../styles";
+
+interface ErrorBannerProps {
+  errors: string[];
+}
+
+const ErrorBanner: React.VFC<ErrorBannerProps> = ({ errors }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  const errorTitle =
+    errors.length === 1
+      ? "1 error in configuration file"
+      : `${errors.length} errors in configuration file`;
+
+  return (
+    <BannerContainer data-cy="configuration-errors-banner">
+      <Callout variant="warning" title={errorTitle.toUpperCase()}>
```

Comment:
Might we not capitalize the title text here? Since it's not capitalized for warnings it seems preferable to standardize